lib/ApiRequestor.php

```diff
@@ -39,6 +44,28 @@ public function __construct($apiKey = null, $apiBase = null)
         $this->_apiBase = $apiBase;
     }
 
+    /**
+     * @static Creates a telemetry json blob for use in 'X-Stripe-Client-Telemetry' headers
+     *
+     * @param RequestTelemetry $requestTelemetry
+     * @return string
+     */
+    private static function _telemetryJson($requestTelemetry)
+    {
+        $payload = array(
+            'last_request_metrics' => array(
+                'request_id' => $requestTelemetry->requestId,
+                'request_duration_ms' => $requestTelemetry->requestDuration,
+        ));
+
+        $result = json_encode($payload);
+        if ($result != false) {
```

Comment:
My concern was that IF something _did_ happen, all we'd get in the header is `false`, since the return is either json or a boolean.  If we had `false` then we'd probably have to special case any analysis tooling in stripe to skip that as we'd only be expecting json.  I can definitely pass it through, but I didn't think it would be helpful.

Reply:
Okay fair enough. Could you add a logger line in the failure case like you did below then? IMO it's best to never paper over errors completely.